ocaml/xapi/xapi_pool.ml

```diff
@@ -3406,21 +3406,33 @@ let enable_tls_verification ~__context =
   | Some self ->
       Xapi_cluster_host.set_tls_config ~__context ~self ~verify:true
 
-let contains_bundle_repo ~__context ~repos =
-  List.exists
-    (fun repo -> Db.Repository.get_origin ~__context ~self:repo = `bundle)
-    repos
-
 let assert_single_bundle_repo_can_be_enabled ~__context ~repos =
```

Comment:
I think we also need to have the same kind of restriction for `remote_pool` repository, or how about change it to: `assert_single_update_repo_can_be_enabled`.